workflows/smc.py

```diff
@@ -40,7 +40,7 @@ def write_smcpp_file(path, output, pop_name, num_sampled_genomes=2, mask_interva
     with open(vcf_file, "w") as vcf:
         ts.write_vcf(vcf, contig_id=chr_name)  # site_mask=np.array(bool)
     # index/compress the vcf
-    cmd = f"bgzip {vcf_file}"
+    cmd = f"bgzip -f {vcf_file}"
```

Comment:
I killed my pipeline in the middle of a run. bgzip baulked since the vcf_files already existed. adding -f ensured the command runs without the dialogue asking if you're sure you want to overwrite the existing file.